### PR TITLE
Keep v3 injected environment variable list

### LINKED ISSUE #2259: tox4: injected environment variables should match with tox 3
```
PATH=/home/bernat/git/pyproject-api/.tox/py39/bin:/home/bernat/.local/bin:/usr/local/bin:/usr/bin:/var/lib/snapd/snap/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
LANG=en_GB.UTF-8
SSL_CERT_FILE=/etc/ssl/cert.pem
TOX_WORK_DIR=/home/bernat/git/pyproject-api/.tox
COVERAGE_FILE=/home/bernat/git/pyproject-api/.tox/.coverage.py39
PYTHONHASHSEED=1547490352
TOX_ENV_NAME=py39
TOX_ENV_DIR=/home/bernat/git/pyproject-api/.tox/py39
TOX_PACKAGE=/home/bernat/git/pyproject-api/.tox/.tmp/package/1/pyproject_api-0.1.dev1+g47f2fa9.d20211022.tar.gz
VIRTUAL_ENV=/home/bernat/git/pyproject-api/.tox/py39
```

vs
```
TERM=xterm-256color
SSL_CERT_FILE=/etc/ssl/cert.pem
LANG=en_GB.UTF-8
PATH=/home/bernat/git/pyproject-api/.tox/4/py39/bin:/home/bernat/.local/bin:/usr/local/bin:/usr/bin:/var/lib/snapd/snap/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
PIP_DISABLE_PIP_VERSION_CHECK=1
COVERAGE_FILE=/home/bernat/git/pyproject-api/.tox/4/.coverage.py39
TOX_PACKAGE=/home/bernat/git/pyproject-api/.tox/4/.pkg/dist/pyproject_api-0.1.dev1+g47f2fa9.d20211022-py3-none-any.whl
COLUMNS=213
LINES=86
```

### PR DESCRIPTION
**Pull request by gaborbernat**
Resolves #2259.

### CODE REVIEW

File: docs/changelog/2259.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Insert ``TOX_WORK_DIR``, ``TOX_ENV_NAME``, ``TOX_ENV_DIR`` and ``VIRTUAL_ENV`` environment variables for each tox
+environments environment variables to keep contract with tox version 3 - by :user:`gaborbernat`.
```
Comment (jugmac00) [inline docs/changelog/2259.bugfix.rst:2]:
```suggestion
Insert ``TOX_WORK_DIR``, ``TOX_ENV_NAME``, ``TOX_ENV_DIR`` and ``VIRTUAL_ENV`` into the environment variables for all tox environments to keep contract with tox version 3 - by :user:`gaborbernat`.
```

Comment (jugmac00) [reply docs/changelog/2259.bugfix.rst:2]: What do you think of the updated sentence? The previous one was a bit hard to understand.

**Review by jugmac00 — APPROVED** [review]

File: docs/changelog/2259.bugfix.rst (hunk 1 above)
Comment (gaborbernat) [reply docs/changelog/2259.bugfix.rst:2]: Sure :+1:

Comment (jugmac00) [reply docs/changelog/2259.bugfix.rst:2]: You are too fast 😎

Comment (gaborbernat) [reply docs/changelog/2259.bugfix.rst:2]: Done in https://github.com/tox-dev/tox/commit/669faa563e4ecf975ce74b9911652e5a1fa37f0c

**Pull request merged**